gamebook/game_manager.py

```diff
@@ -0,0 +1,43 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+
+    def start(self, scene_import):
+        current_scene = self.scene
+        while True:
+            next_scene_name = current_scene.run_scene()
```

Comment:
The variable names here are confusing. Why do you have a `run_scene` method here and also the same name is `Scene`?

Reply:
`run_scene` returns the `next_scene`'s name so that `get_next_scene` could find it at the scene list from the scene file we import